frontend/mulighetsrommet-veileder-flate/src/components/tabs/TiltaksdetaljerFane.tsx

```diff
@@ -12,15 +29,39 @@ const TiltaksdetaljerFane2 = () => {
           <Tabs.Tab key={index} value={`tab${index + 1}`} label={fane} className="btn__tab" />
```

Comment:
Ser at denne ikke er tab-bar slik den er nå. Legg på `tabIndex={0}` så kan man tabbe til de ulike fanene med tastatur.

Reply:
Den skal funke om du bruker piltaster? Det er designsystemet sin komponent

Reply:
Jepp, fått det grundig forklart i #designsystem https://nav-it.slack.com/archives/C7NE7A8UF/p1654760667489139 🤣